__tests__/__snapshots__/plugin.test.js.snap

```diff
@@ -121,9 +121,9 @@ exports[`babel plugin supports SequenceExpression, with objectHook 1`] = `
 function App() {
   (0, useAnimatedGestureHandler)({
     onStart: function () {
-      var _e = [new Error(), 1, -20];
+      const _e = [new Error(), 1, -20];
 
-      var _f = function _f() {};
+      const _f = function () {};
```

Comment:
Previously we had `function _f() {};` but after the change the function has no name. I guess that's okay since `_f` was hard-coded in each worklet and thus gives no additional information that would help to identify them.